src/main.ts

```diff
@@ -185,6 +186,7 @@ async function getTsServerRepoResult(
     newTsServerPath: string,
     downloadDir: string,
     replayScriptArtifactPath: string,
+    rawErrorsScriptArtifactPath: string,
```

Comment:
I'm sure this was my own suggestion, but I think it might be clearer to replace "rawErrorsScript" with just "rawError" (no 's') everywhere, since there's only one error and it's not a script.